util/util.go

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package util contains utility functions and types.
+package util
```

Comment:
we could name the package `value`, so it's `value.IsZero(x)`

Reply:
Agree. Moment.

Reply:
Done!